test/Identity.IntegrationTest/openid-configuration.json

```diff
@@ -38,7 +38,8 @@
     "refresh_token",
     "implicit",
     "password",
-    "urn:ietf:params:oauth:grant-type:device_code"
+    "urn:ietf:params:oauth:grant-type:device_code",
+    "extension"
```

Comment:
note: `extension` will most likely change name to `webauthn` as part of [PM-2032 Build server endpoints to support authentication with a passkey](https://bitwarden.atlassian.net/browse/PM-2032)

Reply:
Yep, but to avoid scope creeeeeeep I opted to get the test to pass and address the new name in the appropriate JIRA task.